src/analysis/bits-lattice.h

```diff
@@ -0,0 +1,83 @@
+#ifndef wasm_analysis_bits_lattice_h
+#define wasm_analysis_bits_lattice_h
+
+#include "lattice.h"
+
+namespace wasm::analysis {
+
+// Lattice for used bits analysis. A flat uint32_t lattice, where
+// all numerical values are equal to each other in an abstract
+// sense, and there is a top above and a bottom below them.
+struct BitsLattice {
+  enum Status { BOTTOM, NUMBER, TOP };
+
+  class Element {
+    Status status = Status::BOTTOM;
+    uint32_t value = 0;
```

Comment:
Why can't we just use `value`, in which top and bottom would be all zeros or all ones?